docs/howto/operate/add-aws-hub.md

```diff
@@ -0,0 +1,222 @@
+# Add a new hub in a AWS kops-based cluster
+
+The idea behind this guide is showcase the process of setting up a 
+[kops](https://kops.sigs.k8s.io/getting_started/aws/)-based AWS
+cluster and manually deploy a new hub on top of it using our deployer tool.
+This is a preliminary but fully functional and manual process. Once
+[#381](https://github.com/2i2c-org/pilot-hubs/issues/381) is resolved, we should be able
+to automate the hub deployment process as we currently do with the GKE-based hubs.
+
+```{note}
+We will continue working toward a definitive one once we figured out some of the
+discussions outlined in [#431](https://github.com/2i2c-org/pilot-hubs/issues/431).
+```
+
+## Pre-requisites
+
+1. Follow the instructions outlined in
+[Set up and use the the deployment scripts locally](./manual-deploy.md) to set up the
+local environment and prepare `sops` to encrypt and decrypt files.
+
+2. Install the awscli tool (you can use pip or conda to install it in the environment)
+and configure it to use the provided AWS user credentials. [Follow this guide](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config)
+for a quick configuration process.
+
+```{note}
+The customer with AWS admin privileges should have created a user for you with full
+privileges. We will probably explore fine-graining the permissions actually needed
+in the short-term.
+```
+
+3. Export some helpful AWS environment variables (because "aws configure" doesn't export
+these environments vars for kops to use, so we do it manually).
+
+```bash
+export AWS_PROFILE=<cluster_name>
+export AWS_ACCESS_KEY_ID=$(aws configure get aws_access_key_id)
```

Comment:
I think an easier / safer way is to put these in `~/.aws/credentials` and just export `AWS_PROFILE`. Does `aws configure` populate `~/.aws/credentials`?

Reply:
> Does aws configure populate ~/.aws/credentials?

Yep, it does. In fact, those are already in the credentials file (this is why you can get them with `aws configure get`).
But AFAIK, when you export the profile name, the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY env variables are not exported as well and kops seems to need those accordingly to the docs (check the last box at that section): https://kops.sigs.k8s.io/getting_started/aws/#setup-iam-user.

Reply:
Since this is minor, I will not block the merge on it.
We can always iterate and change it later down the road.
Btw, feel free to continue the discussion.